include/util/dist_table_wrapper.hpp

```diff
@@ -46,6 +46,30 @@ template <typename T> class DistTableWrapper
         return table_[index];
     }
 
+    void InvalidateRoute(NodeID from, NodeID to)
```

Comment:
Should these functions be part of the `DistTableWrapper`? This class seems very generic and the functions are only relevant in the context of the `trip` plugin.

Another way to do it would be to have two functions `invalidateRoute(DistanceTableWrapper &dist, NodeID from, NodeID to)` and `shortcutRoute(DistanceTableWrapper &dist, NodeID from, NodeID to)` in the trip plugin.